src/main/java/randoop/sequence/VariableRenamer.java

```diff
@@ -22,32 +25,37 @@ public VariableRenamer(Sequence sequence) {
    * Heuristically transforms variables to better names based on its type name. Here are some
    * examples:
    *
+   * <p>
+   *
    * <pre>
-   *   int var0 = 1     becomes  int i0 = 1
+   *   int var0 = 1     becomes  int int0 = 1
    *   ClassName var0 = new ClassName()      becomes ClassName className = new ClassName()
-   *   Class var0 = null      becomes Class clazz = null
+   *   Class var0 = null      becomes Class cls = null
    * </pre>
    *
    * @param type the type to use as base of variable name
    * @return a variable name based on its type, without a trailing number
    */
   static String getVariableName(Type type) {
+    return getVariableName(type, 0);
+  }
 
+  static String getVariableName(Type type, int depth) {
```

Comment:
Please write Javadoc documentation for every method.
(For example, here the `depth` parameter is not obvious.)